packages/opentelemetry-exporter-zipkin/src/zipkin.ts

```diff
@@ -64,12 +64,12 @@ export class ZipkinExporter implements SpanExporter {
     spans: ReadableSpan[],
     resultCallback: (result: ExportResult) => void
   ) {
-    if (typeof this._serviceName !== 'string') {
-      this._serviceName = String(
+    const serviceName = String(
```

Comment:
At first I thought you are removing the service name from config (as you removed that everywhere from collector exporter) but then here you still leave it, this looks for me like some inconsistency across exporters, why is it removed from exporter collector and here it is handled differently.  So service name can be still set by using config in zipkin but this is not possible with collector exporter

Reply:
I removed it from the collector exporter because the collector is an official opentelemetry component and should use the official mechanism. The proto/collector also doesn't have a specific field for service name and just treats it as any other resource property.

I left it in zipkin and jaeger because they have specific service name properties in their protocols and their users may be used to configuring them that way. It also allows those users to override the resource if they want.

Reply:
@obecny if that is an issue would you rather I bring back the serviceName config in collector or remove it from zipkin?

Reply:
No strong opinion here tbh just thought at first it should be somehow unified across exporters so either keep in all or remove and always take from resource. I wonder what other sigs are doing with regards to service name name then. If they unified this or have different behaviour for each exporter.

Reply:
Updated jaeger to have this same behavior (will try to get it from the first resource unless the user specifically configures exporter)

I think because the collector exporter is our first party, it should use the official method (resource).